modules/ROOT/pages/policy-mule4-jwt-validation.adoc

```diff
@@ -153,8 +147,11 @@ Note that all DataWeave expressions must return a boolean value or they will alw
 |===
 
 == Obtaining a policy compatible PEM public key
-The policy uses public rsa key in pem format. In this section we explain how to obtain the public.pem file that contains said key.
-Remember that once the public.pem key is obtained, before pasting it to the policy configuration to remove the *-----BEGIN PUBLIC KEY-----* header and the *-----END PUBLIC KEY-----* footer.
+
+The policy uses public rsa key in pem format. 
+After you obtain the public.pem key, remove the *-----BEGIN PUBLIC KEY-----* header and the *-----END PUBLIC KEY-----* footer before you paste it to the policy configuration.
+
+*Eva--why must the customer perform these steps and when should they do this?*
```

Comment:
People always ask about this maybe we can put a note at the beginning of this doc telling that this exists down here.
this section is only usefull if the JWT Signing Method == RSA
The.y will perform this when they have a pre-existing RSA key that they want to use and is in a format that we don't support. Most of them don't know how to convert them to the format we use in the policy.

Reply:
Added the content. Done! Thanks!